Manufacturers.md

```diff
@@ -85,4 +85,4 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |VIVA|VivaFPV|https://vivafpv.com/|
 |WIZZ|WIZZ|https://wizz.store/|
 |YUPF|YuPiFc|https://www.yupifc.com/|
-|ZEEZ|ZeeZ Design|http://www.zeezdesign.com/|
+|ZEEZ|ZeeZ RC|http://www.zeezrc.com/|
```

Comment:
This is not editable by manufacturers - if you would like a change please open an issue asking for it.

Reply:
OK opened the issue